src/utils/testData.js

```diff
@@ -0,0 +1,171 @@
+    clicks: 185,
+    impressions: 8760
+  },
+  {
+    campaign: "AT|SN|Snowboard|Generisch",
+    channel: "Search",
+    clicks: 203,
+    impressions: 5966
+  },
+  {
+    campaign: "DE | SP Gadgets",
+    channel: "Search",
+    clicks: 897,
+    impressions: 15324
+  },
+  {
+    campaign: "DE|SN|Snowboard|Brands",
+    channel: "Search",
+    clicks: 121,
+    impressions: 3002
+  },
+  {
+    campaign: "EN | SP Gadgets",
+    channel: "Search",
+    clicks: 259,
+    impressions: 6738
+  },
+  {
+    campaign: "AT|SN|Skullcandy",
+    channel: "Search",
+    clicks: 76,
+    impressions: 712
+  },
+  {
+    campaign: "AT|SN|Longboard|Generisch",
+    channel: "Search",
+    clicks: 189,
+    impressions: 2683
+  },
+  {
+    campaign: "AT|SN|Audio&Video|Brands",
+    channel: "Search",
+    clicks: 23,
+    impressions: 800
+  },
+  {
+    campaign: "AT|SN|Skate|Generisch",
+    channel: "Search",
+    clicks: 476,
+    impressions: 6124
+  },
+  {
+    campaign: "AT|SN|Snowboard|Brands",
+    channel: "Search",
+    clicks: 107,
+    impressions: 995
+  },
+  {
+    campaign: "DE|SN|Accessoires|Brands",
+    channel: "Search",
+    clicks: 156,
+    impressions: 1699
+  },
+  {
+    campaign: "DE|SN|Skullcandy",
+    channel: "Search",
+    clicks: 664,
+    impressions: 5533
+  },
+  {
+    campaign: "AT|SN|Skate|Brands",
+    channel: "Search",
+    clicks: 44,
+    impressions: 671
+  },
+  {
+    campaign: "DE|SN|Longboard|Generisch",
+    channel: "Display",
+    clicks: 9,
+    impressions: 141
+  },
+  {
+    campaign: "AT|SN|Snowboard|Generisch",
+    channel: "Display",
+    clicks: 798,
+    impressions: 19014
+  },
+
+  {
+    campaign: "DE | SP Gadgets",
+    channel: "Display",
+    clicks: 295,
+    impressions: 3761
+  },
+  {
+    campaign: "DE|SN|Snowboard|Brands",
+    channel: "Display",
+    clicks: 292,
+    impressions: 98933
+  },
+  {
+    campaign: "EN | SP Gadgets",
+    channel: "Display",
+    clicks: 28,
+    impressions: 143
+  },
+  {
+    campaign: "AT|SN|Skullcandy",
+    channel: "Display",
+    clicks: 122,
+    impressions: 1138
+  },
+  {
+    campaign: "AT|SN|Longboard|Generisch",
+    channel: "Display",
+    clicks: 4064,
+    impressions: 164954
+  },
+  {
+    campaign: "AT|SN|Audio&Video|Brands",
+    channel: "Display",
+    clicks: 912,
+    impressions: 76522
+  },
+  {
+    campaign: "AT|SN|Skate|Generisch",
+    channel: "Display",
+    clicks: 89,
+    impressions: 4471
+  },
+  {
+    campaign: "AT|SN|Snowboard|Brands",
+    channel: "Display",
+    clicks: 652,
+    impressions: 27188
+  },
+  {
+    campaign: "DE|SN|Accessoires|Brands",
+    channel: "Display",
+    clicks: 146,
+    impressions: 389
+  },
+  {
+    campaign: "DE|SN|Skullcandy",
+    channel: "Display",
+    clicks: 312,
+    impressions: 642
+  }
+];
+
+const option1 = "DE|SN|Longboard|Generisch";
+const option2 = "Search";
+const option3 = "AT|SN|Snowboard|Generisch";
+// const option4 = "DE | SP Gadgets";
```

Comment:
either use option 4 to 15 or remove comments

Reply:
9f87f40e9771eba9a654b1f4fc1f595280510a3f